app/views/routes/_base_form.html.haml

```diff
@@ -1,4 +1,4 @@
-= bootstrap_form_for([project, route]) do |f|
+= bootstrap_form_for([project, route], url: if route.new_record? then nil else "/projects/#{project.id}/routes/#{route.id}?section=base" end) do |f|
```

Comment:
Explained in my commit message:

```
This work bring up a latent error:
when the user gives an invalid information, the form disappeared.
The reason was the shown form was depending on query parameter `section`.
However, when updating the route, this information was not given.
```

Reply:
Oh good point. I tried to find a better way to fix it but didn't find. Just a suggestion to use route_helper rather than hardcoded string. 

`= bootstrap_form_for([project, route], url: if route.new_record? then nil else project_route_path(project, route, section: :base) end) do |f|`